src/OpenTelemetry.Exporter.Prometheus.Shared/PrometheusExporterOptions.cs

```diff
@@ -15,61 +15,25 @@
 // </copyright>
 
 using System;
-using System.Collections.Generic;
 using OpenTelemetry.Internal;
 
-namespace OpenTelemetry.Exporter
+#if PROMETHEUS_ASPNETCORE
+namespace OpenTelemetry.Exporter.Prometheus.AspNetCore
+#elif PROMETHEUS_HTTPLISTENER
+namespace OpenTelemetry.Exporter.Prometheus.HttpListener
+#else
+namespace OpenTelemetry.Exporter.Prometheus.Shared
+#endif
 {
     /// <summary>
-    /// <see cref="PrometheusExporter"/> options.
+    /// Prometheus exporter options.
```

Comment:
the ns already has prometheus in it, so class name again having prometheus is bit looong.